js/ui/components/clubhouse.js

```diff
@@ -279,9 +297,11 @@ var ClubhouseQuestBanner = new Lang.Class({
     Name: 'ClubhouseQuestBanner',
     Extends: ClubhouseNotificationBanner,
 
-    _init: function(notification) {
+    _init: function(notification, isFirstBanner) {
```

Comment:
It would be clearer to call this `shouldSlideIn` instead of `isFirstBanner` -- the banner class itself does not need to know whether it was the first banner or not, only if it should slide.

Reply:
I did this on purpose so we associated the first-banner -> slide-in inside the banner itself. But I can change it.

Reply:
I decided to keep it in the end. It makes sense in my mind to make the distinction as it can also be better in the future if we need to change this slide in effect by something else.